Appsilon/shiny.benchmark
# Make devtools::check to run smoothly

## Changes description

`devtools::check()` is currently raising errors, warnings and notes. To make it easy to fix the next problems we will face, it is important to fix the current ones. In task #6 the DESCRIPTION file will be improved which means that right now we only want `devtools::check()` to run smoothly.

## Related issues

close #17

## How to test it

Run `devtools::check()`. The desired output is as follow:

![Screenshot from 2022-10-18 13-35-06](https://user-images.githubusercontent.com/13513569/196421268-9ce9e5f6-f340-41c7-bc66-1b63a60c8444.png)
